lib/api/helpers/application_tokens.rb

```diff
@@ -22,10 +22,10 @@ def create_application_token!(user, id, params)
             status 201
             { plain_token: plain_token }
           else
-            bad_request!(application_token.errors)
+            unprocessable_entity!(application_token.errors)
```

Comment:
I was wondering... we've been using `model.errors.full_messages` around. Should we use `errors` that keeps indexed by the attribute or should we use `full_messages`?

I should have mentioned this in the other PR but...

Reply:
That's something we need to review, correct :sweat_smile:  We should come up with a uniform way of reporting errors (these helper methods are a first step). I'll take a look and come back with what I found :+1: